setupbase.py

```diff
@@ -102,8 +104,8 @@ def finalize_options(self):
                 'Local zipfile does not exist: %s' % self.zip_path)
 
     def __print(self, msg):
-        """Shortcut for printing with the distutils logger."""
-        self.announce(msg, level=distutils.log.INFO)
+        """Shortcut for printing INFO message with the setuptools (or distutils) logger."""
+        self.announce(msg, level=2)
```

Comment:
If there are no constants for the log level when using `setuptools` maybe we should define them here? Something at the top of the file like:

```python

LOG_INFO = 2

```

Reply:
I agree, setuptools seems not to provide the constant, so I will add LOG_INFO = 2